photonpump/connection.py

```diff
@@ -1106,6 +1106,41 @@ def __init__(self, connector, dispatcher, credential=None):
     async def __aexit__(self, exc_type, exc, tb):
         await self.close()
 
+    async def find_backwards(
+            self,
+            stream: str,
+            predicate: Callable[[Event], bool],
+
+    ):
+        """
+        Return first event matching predicate or None in none exists.
+
+        Args:
+            stream: The name of the stream.
+            predicate: Callable which get's an event as parameter and return
+            boolean as matching result
+
+        Examples:
+
+            Find event with event.type 'my-event-desired' in stream
+            'my-stream-name'.
+
+            >>> with async.connect() as conn:
+            >>>     event = await conn.find_backwards(
+            >>>         'my-stream-name',
+            >>>         predicate=lambda event: 'my-event-desired' in event.type
+            >>>     )
+        """
+
+        async for event in self.iter(
+            stream,
+            batch_size=10,
```

Comment:
I think probably this should stay in the signature with the default=10, rather than hardcoded in here. What do you @palankai @epoplavskis think about that?

Reply:
fixed